utils/releaser/src/apple/core.rs

```diff
@@ -11,7 +11,7 @@ static MAC_LIB_DIR: &str = "clients/apple/CLockbookCore/Sources/CLockbookCore/li
 static LIB: &str = "liblockbook_core.a";
 static HEAD: &str = "lockbook_core.h";
 
-pub fn build() {
+pub fn build_apple() {
```

Comment:
why? The full path of this function is `apple::core::build`, so you don't need to specify `apple`.